.github/workflows/test.yml

```diff
@@ -47,4 +47,4 @@ jobs:
               run: |
                 export PATH=$PATH:${{ steps.get_smc_storm.outputs.SMC_STORM_PATH }}
                 source /opt/ros/${{ matrix.ros-distro }}/setup.bash                
-                pytest-3
\ No newline at end of file
+                pytest-3 -vss
```

Comment:
Is the double `s` a typo? Cannot find that flag in the docu!

Reply:
Good point. I have no idea where this came from